filter/data.go

```diff
@@ -21,7 +21,7 @@ type Dimensions struct {
 // Dimension represents a dimension response from the filter api
 type Dimension struct {
 	Name string `json:"name"`
-	URI  string `json:"dimension_url"`
+	URI  string `json:"dimension_url,omitempty"`
```

Comment:
Why did we change it?
Are we absolutely sure this won't be required when serializing?

Reply:
sorry, let me make it unchanged

Reply:
@PauloPortugal i already commited a fix just now with "-S", help to check if the GPG key sign works

Reply:
Your first commit is still showing as being created by `root`. you need to fix that.